get-poetry.py

```diff
@@ -814,9 +814,14 @@ def call(self, *args):
         return subprocess.check_output(args, stderr=subprocess.STDOUT)
 
     def _get(self, url):
+        """Perform a GET request through Python to a specific URL
+
+        The URL connection has a specific timeout set to 2 seconds
+        to prevent indefinite hanging in certain edge cases
+        """
         request = Request(url, headers={"User-Agent": "Python Poetry"})
 
-        with closing(urlopen(request)) as r:
+        with closing(urlopen(request, timeout=2)) as r:
```

Comment:
This is a good idea but should probably support `POETRY_GET_TIMEOUT` or something with only a _default_ of this value. Also the default value should probably be 60s +